### ISSUE TITLE
Error messages when converting from Jekyll

### ISSUE THREAD

**Issue by ChristopherBiscardi** [ChristopherBiscardi/gatsby-mdx#273]
in https://github.com/ChristopherBiscardi/gatsby-mdx/issues/63#issuecomment-457764281 @dceddia writes:

I ran into a related problem while trying to import a bunch of blog posts from a Jekyll site. There are formatting issues I need to fix, but the errors don't say which file caused them, so it's a game of grepping for unique-looking strings and hoping for the best 😅 

My hacky workaround was to wrap most of the code in `on-create-node.js` in a try/catch and prepend the filename to the error message:

```js
try {

  /**** Everything in this block is original to the file, no changes here ***/

  const mdxNode = await createMDXNode({
    id: createNodeId(`${node.id} >>> Mdx`),
    node,
    content
  });

  createNode(mdxNode);
  createParentChildLink({ parent: node, child: mdxNode });

  // write scope files into .cache for later consumption
  const { scopeImports, scopeIdentifiers } = await genMDX({
    node: mdxNode,
    getNode,
    getNodes,
    reporter,
    cache,
    pathPrefix,
    options
  });
  await cacheScope({
    scopeIdentifiers,
    scopeImports,
    createContentDigest: contentDigest,
    directory: store.getState().program.directory,
    parentNode: node
  });
} catch (e) {
  // Add the filename to the error and rethrow
  e.message = `in ${node.absolutePath} -- ${e.message}`;
  throw e;
}
```

I don't know much about Gatsby at this point so I'm sure there's a more proper way to do this, but it did help me track down the errors at least.

**Comment by ChristopherBiscardi** [comment]
One debugging tip is to run

```shell
DEBUG=gatsby* yarn develop
```

**Comment by johno** [comment]
This is fixed by #296
